packages/build-info/src/frameworks/nuxt.ts

```diff
@@ -28,15 +28,19 @@ export class Nuxt extends BaseFramework implements Framework {
     await super.detect()
 
     if (this.detected) {
-      // Use Nuxt 3+ config as default, only apply legacy config for versions < 3
-      if (this.detected.package?.name === 'nuxt3' || (this.detected.package?.version?.major !== undefined && this.detected.package.version.major >= 3)) {
-        // Nuxt 3+ config (including v4 and future versions)
-        this.name = 'Nuxt 3'
-        this.build.command = `nuxt build`
-        this.build.directory = `dist`
-        this.dev.command = `nuxt dev`
+      // Only use modern config for nuxt3 package or nuxt/nuxt-edge with explicit version >= 3
+      // Default to legacy for nuxt/nuxt-edge without version info
+      if (
+        this.detected.package?.name === 'nuxt3' ||
+        (this.detected.package?.version?.major !== undefined && this.detected.package.version.major >= 3)
```

Comment:
This comment doesn't match the implementation. We ONLY use the legacy for a major version <3. That's it. Don't implement or explain anything else.

(`nuxt3` has a major version of 3 so it's not a special case)